plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -932,10 +952,38 @@ public void createTable(ConnectorSession session, ConnectorTableMetadata tableMe
 
         try {
             TrinoFileSystem fileSystem = fileSystemFactory.create(session);
-            if (!fileSystem.listFiles(deltaLogDirectory).hasNext()) {
+            boolean transactionLogFileExists = fileSystem.listFiles(deltaLogDirectory).hasNext();
+
+            if (!replaceExistingTable && transactionLogFileExists) {
+                throw new TrinoException(
+                        NOT_SUPPORTED,
+                        "Using CREATE TABLE with an existing table content is disallowed, instead use the system.register_table() procedure.");
```

Comment:
Maybe we should hint the possibility of using `CREATE OR REPLACE TABLE` as well in the exception message.